src/docker/Dockerfile

```diff
@@ -69,6 +106,13 @@ RUN pipx install pre-commit==${PRE_COMMIT_VERSION}
 ARG AWS_CLI_VERSION=1.16.287
 RUN pipx install awscli==${AWS_CLI_VERSION}
 
-USER root
+#set helm aliases
+ARG HELMDIFF_HELM2_VERSION=v2.11.0+5
+ARG HELMDIFF_HELM3_VERSION=v3.0.0-rc.7
+RUN helm init --client-only && \
+    helm plugin install https://github.com/databus23/helm-diff --version ${HELMDIFF_HELM2_VERSION} && \
```

Comment:
Can this go in the helm2 install block? Or right after it?

Reply:
This needs to be done with the dadsgarage user I can switch the user or should init it here what would you prefer?

Reply:
Oh okay. Didn't realize that. It's okay where it is, though I think it would be better to separate them out into "helm 2 initialization and plugin installation" and "helm 3 plugin installation".

It will add an additional layer to the docker image, but i think it will be cleaner in the docker file, and we can come along and add more helm2 or helm2 plugins later, and there will be a place for them to go.